index.html

```diff
@@ -135,6 +135,36 @@ <h4>EditContext state</h4>
               from the [=Text Input Service=]. The user agent will indicate which positions are
               required by firing {{CharacterBoundsUpdateEvent}}.
             </p>
+            <p>
+              [=Control bounds=], [=selection bounds=], and [=codepoint rects=] are given in the
+              <dfn>client coordinate system</dfn>, which is defined as a two-dimensional Cartesian
+              coordinate system (x, y) where the origin is the top-left corner of the
+              <a href="https://drafts.csswg.org/cssom-view/#layout-viewport">layout viewport</a>,
+              the x-axis points towards the top-right of the
+              <a href="https://drafts.csswg.org/cssom-view/#layout-viewport">layout viewport</a>,
+              and the y-axis points towards the bottom-left of the
+              <a href="https://drafts.csswg.org/cssom-view/#layout-viewport">layout viewport</a>.
+              The units of the client coordinate system are
+              <a href="https://drafts.csswg.org/css-values/#px">CSS pixels</a>.
+            </p>
+            <div class="note">
+              <p>
+                Since EditContext bounds are defined in
+                <a data-lt="client coordinate system">client coordinates</a>, the coordinates
+                indicating a given piece of content on a page will change as the user scrolls the
```

Comment:
> the coordinates indicating a given piece of content on a page will change as the user scrolls

Does Browser Zoom and Panning affect these coordinates as well or it can deduced from the layout viewport definition? Note that panning doesn't change the layout viewport so I'm guessing the coordinates would remain unchanged.

Reply:
Based on the definition of the layout viewport (and experimenting on my part with `getBoundingClientRect()`), I believe browser zoom/panning does not affect these coordinates (although browser zoom can cause content to move or be laid out differently on the page). So I think it is OK to just lean on the layout viewport definition for this.

Reply:
Ack.